src/components/WalletConnect.vue

```diff
@@ -106,12 +106,19 @@ import {
 } from "@algo-builder/web";
 import WalletStore from "../store/WalletStore";
 import { searchApplicationAndAccount } from "@/indexer";
-import { walletMessage } from "../constants/constant";
+import {
+	MAIN_NET_URL,
+	BETA_NET_URL,
+	TEST_NET_URL,
+	walletMessage,
+} from "../constants/constant";
 import DaoID from "@/store/DaoID";
 import {
 	openErrorNotificationWithIcon,
 	WALLET_CONNECTION_ERROR,
 } from "@/constants";
```

Comment:
You have tried to use another import from same folder. `"../constants/constant"` and `@/constants`. Please use import. I would recommend 2nd one.

Reply:
yeah you are right, I missed it, will update